python/ray/train/gbdt_trainer.py

```diff
@@ -59,9 +60,38 @@ def _convert_scaling_config_to_ray_params(
             "num_actors": int(num_actors),
         }
     )
-    ray_params = ray_params_cls(
+
+    # This should be upstreamed to xgboost_ray,
+    # but also left here for backwards compatibility.
+    if not hasattr(ray_params_cls, "placement_strategy"):
+
+        class RayParamsFromScalingConfig(ray_params_cls):
+            placement_strategy: Optional[str] = None
```

Comment:
```suggestion
            placement_strategy: str = "PACK"
```
it currently defaults to PACK in xgboost-ray, so this should probably be reflected here